spring-reactive-modules/spring-reactive-3/src/main/java/com/baeldung/custom/deserialization/model/OrderResponse.java

```diff
@@ -0,0 +1,68 @@
+package com.baeldung.custom.deserialization.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+
```

Comment:
we can use lombok

Reply:
Thanks for comment, in my previous PRs i got a comment not to use Lombok unless the topic itself is related to Lombok so not sure about the expectation in PR

Reply:
No problem then